iphone/Classes/TiUtils.m

```diff
@@ -201,6 +205,15 @@ +(BOOL)isIOS9_1OrGreater
     return [UITouch instancesRespondToSelector:@selector(altitudeAngle)];
 }
 
++(BOOL)isIOS9_3OrGreater
+{
+#if IS_XCODE_7_3
+    return [WCSession instancesRespondToSelector:@selector(activationState)];
+#else
+    return NO;
+#endif
```

Comment:
You can do , it will return a String with the current iOS version running on the device.  [[UIDevice currentDevice] systemVersion];